.github/workflows/continuous-integration.yml

```diff
@@ -23,24 +23,24 @@ jobs:
           - "7.4"
 
     steps:
-      - name: "Checkout"
-        uses: "actions/checkout@v2.0.0"
+      - name: "Checkout code"
+        uses: "actions/checkout@v2"
 
       - name: "Install PHP"
-        uses: "shivammathur/setup-php@1.8.1"
+        uses: "shivammathur/setup-php@v2"
         with:
-          coverage: "pcov"
+          coverage: "none"
```

Comment:
does this mean that we do not have a coverage metric for this library?

Reply:
Yes, we don't have one.

Reply:
👍